test/System/Time.cpp

```diff
@@ -163,7 +161,7 @@ TEST_CASE("sf::Time class - [system]")
             CHECK(sf::seconds(1) / static_cast<sf::Int64>(2) == sf::seconds(0.5f));
             CHECK(sf::seconds(42) / static_cast<sf::Int64>(2) == sf::seconds(21));
             CHECK(sf::seconds(1) / sf::seconds(1) == 1.0f);
-            CHECK(sf::milliseconds(10) / sf::microseconds(1) == Approx(10'000.0).epsilon(1e-6));
+            CHECK(sf::milliseconds(10) / sf::microseconds(1) == doctest::Approx(10'000.0).epsilon(1e-6));
```

Comment:
Do you think it would make sense to add `Approx<float>::epsilon()`? E.g. as specialization, or even as method in the general template, which is simply not invoked for non-floats...

We are likely going to use this here and there...

Reply:
Eventually we might. With enough time I could eliminate the need for `.epsilon` here. I'm using it probably because of the 32bit MinGW build using especially low precision floats. If I change the values we're testing then I can eventually produce a result that is close enough to just use `Approx(10'000.f)`.